src/pages/Swap/index.tsx

```diff
@@ -68,7 +69,10 @@ const StyledInfo = styled(Info)`
   }
 `
 
+const ARBITRUM_LOCAL_STORAGE_DISMISSED_KEY = 'HIDE_ARBITRUM_SWAP_ALERT'
```

Comment:
please use redux for storing this state, for which the user slice is already stored to localstorage